Meta/build-root-filesystem.sh

```diff
@@ -137,6 +137,10 @@ chown -R 100:100 mnt/home/anon
 chown -R 200:200 mnt/home/nona
 echo "done"
 
+
+git clone file://"$SERENITY_ROOT" --depth 1 mnt/usr/share/serenity/
+chown -R 100:100 mnt/usr/share/serenity
```

Comment:
Yeah, it does..
Something else we could do is copy the sources at build time with something similar to `serenity_install_headers`, which would allow us to only copy what we need at each build.
Or we could stay with git clone but make it opt-in with some flag.

Reply:
Both of those sound like improvements. Either one is fine, as long as it doesn't chonk up the build time too much

Reply:
Okay, I went with adding `serenity_install_sources` to CMake, build time doesn't seem to be noticeably affected.